scripts/util.py

```diff
@@ -1984,6 +1876,191 @@ def nodeset_map(self, hostnames: list):
 
 lkp = Lookup(cfg)
 
+# Needs to be run after the lookup is complete to get endpoint versions
```

Comment:
We could avoid this code moving by simply removing `compute` global var and using function `def compute()`.

Reply:
I don't think this works.  The intention for the compute is it to be a shared api caller in util.  If we use a function it will create a new caller each time which would be different behavior.

Reply:
I think I fixed this, but I still think we need a global (at the very least, my fix shouldn't break how it works).